### ISSUE TITLE
Measure repeat should be addable by clicking (with warning dialogue, same as click/drag)

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#30775]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

1. Drag a measure repeat on pre-existing music, the warning dialogue appears and you are able to add the measure repeat (replacing the existing music
2. This does not work when selecting the measure and clicking on the palette panel, that only works when adding to an empty measure

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/c98c35bd-eb4f-4640-afbb-734f7356e82c

### In which versions of MuseScore Studio is this issue present?

OS: macOS Sequoia (15.6), Arch.: arm64, MuseScore Studio version (64-bit): 4.7.0-253000329, revision: github-musescore-musescore-d771b8a

### Regression

No.

### Operating system

MacOS15

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
